crates/core/component/governance/src/proposal.rs

```diff
@@ -95,6 +106,11 @@ impl TryFrom<pb::Proposal> for Proposal {
                     commit: if signaling.commit.is_empty() {
                         None
                     } else {
+                        // Commit hash has max length of 64 chars:
+                        if signaling.commit.len() > 64 {
```

Comment:
How about we make this 255 so it can be used with IPFS links, fine to leave as is too